project/Dependencies.scala

```diff
@@ -204,13 +204,12 @@ object Dependencies {
   )
 
   val HBase = {
-    val hbaseVersion = "1.2.6.1"
-    val hadoopVersion = "2.5.2"
+    val hbaseVersion = "1.4.9"
+    val hadoopVersion = "2.7.4"
     Seq(
       libraryDependencies ++= Seq(
         // for some reason version 2.2.3U1 started to get picked which was not accepted by Whitesource)
         "com.sun.xml.bind" % "jaxb-impl" % "2.2.3-1", // CDDL + GPLv2
-        // TODO: remove direct dependency ^^ when updating from these very old versions
```

Comment:
Can the direct `jaxb-impl` dependency go away now?

Reply:
I removed the `jaxb-impl` dependency. HBase tests are OK, but UnixDomainSocketSpec is failing somehow.

Reply:
No worries about UnixDomainSocket.

Does it still depend on `jaxb-impl`? What version?

Reply:
HBase 1.4.9 includes jaxb-impl 2.2.3-1